matlab/schemas/+anatomy/Area.m

```diff
@@ -0,0 +1,13 @@
+%{
+# LUT for Brain Areas
+brain_area      :  enum('V1','P','POR','PM','AM','A','RL','AL','LI','LM')      # area name
+---
+%}
+
+classdef Area < dj.Lookup
+	methods
+		function self = Area(varargin)
```

Comment:
Why do we need a restriction in the constructor?

Reply:
When I wanted to use Area('brain_area="V1"')

Reply:
```matlab
Area & 'brain_area="V1"
```